src/test/java/net/snowflake/client/jdbc/ConnectStringParseTest.java

```diff
@@ -18,13 +18,13 @@ public void testParseAccountName() throws SnowflakeSQLException {
         cstring.getParameters().get(SFSessionProperty.ACCOUNT.getPropertyKey().toUpperCase()),
         is("abc"));
 
-    // Hostname should remain unchanged by default.
```

Comment:
Is this a new bug in our test?

Reply:
This is not part of my PR. Today again when I tried to do git push, if failed. So I did "git pull origin my_branch" and it got other commits in this PR. Before doing pull, I tried "git rebase" but still git push was failing.